### PR TITLE
Contact field priority

### PR DESCRIPTION
**Pull request by dodobas**

### CODE REVIEW

File: templates/contacts/contactfield_managefields.haml [hunk 1]
```diff
@@ -6,31 +6,111 @@
...
-
-        -for field_fields in contact_fields
+        %thead
           %tr
-            %td.field-label
-              {% render_field field_fields.label %}
+            %th.drag-icon
+            %th Name
+            %th Type
+            %th Show
```
Comment (norkans7) [inline templates/contacts/contactfield_managefields.haml]: table header for priority?

**Comment by nicpottier** [comment]
@dodobas and I played with rollover changing background color and it doesn't look good, the rows here are a lot bigger and it just feels odd.

**Comment by ericnewcomer** [comment]
k, will take your word for that, still would like to see dragging to work across the entire grabbable area instead of just the little icon.

**Comment by ycleptkellan** [comment]
You might also play w/ a larger drag handle icon since the checkbox control is so large in comparison.

**Comment by nicpottier** [comment]
I would maybe suggest in cases where there's bike shedding with UI, the UI czars should go sit down with the PR author and work through suggestions until they are happy. Would save a lot of back and forth and fruitless CSS hackery.

**Comment by ericnewcomer** [comment]
FWIW, the pixel pushing stuff (which I think is what you would classify as bike-shedding) I just did my self and committed. Thought the dragging thing was straight forward, but happy look over a shoulder.

**Comment by ericnewcomer** [comment]
Sure, always willing to look over a shoulder while it's around! @dodobas let me know when/if you want extra eyes there.

**Pull request merged**